src/server_manager/electron_app/build.action.sh

```diff
@@ -70,7 +70,11 @@ cp -r "${BUILD_DIR}/server_manager/web_app/static" "${STATIC_DIR}/server_manager
 # is loaded via a custom protocol.
 cp src/server_manager/package.json package-lock.json "${STATIC_DIR}"
 cd "${STATIC_DIR}"
-sed -i "s/0.0.0-debug/${VERSION_NAME}/g" package.json
+if [[ "${OSTYPE}" == "darwin"* ]]; then
```

Comment:
It might be easier to write a Node.js one-liner kind of thing :-)

Reply:
By the way, you don't need the in-place editing. You are copying the file.
You may as well make sed read the original and write to the destination.